larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -519,12 +517,21 @@ INSTANTIATE_TEST_SUITE_P(
     // WARNING: ::testing::Combine accepts max 10 arguments!!!
     ::testing::Combine(
         ::testing::Values(1),  // batches
-        ::testing::Values(std::array<int, 2>{7, 7},
-                          std::array<int, 2>{8, 5}),  // input height/width
-        ::testing::Values(1, 64, 130),                // input depth
-        ::testing::Values(std::array<int, 2>{1, 1}, std::array<int, 2>{3, 3},
-                          std::array<int, 2>{2, 3}),  // filter height/width
-        ::testing::Values(1, 4, 64),                  // filter count
+        ::testing::Values(std::array<int, 3>{7, 7, 1},
```

Comment:
see [here](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#how-to-write-value-parameterized-tests) to create generators for your input arguments. you can pass the generators to the `::testing::Combine` to create tuples with them.

Reply:
Do you mean we can nest the `Combine` call?

Reply:
> Do you mean we can nest the `Combine` call?

I mean you can pass a `generator` instead of an `std::array` to `Combine`

Reply:
@arashb I don't know how to do this, I think it might be easiest if you make a commit on top of this branch/open a PR into this branch with the changes you have in mind?